app/controllers/reports_controller.rb

```diff
@@ -2,6 +2,8 @@ class ReportsController < ApplicationController
   active_tab "reports"
 
   require_permission :can_view_reports?
+  require_permission :can_view_donations?, only: [:net_suite_donation_export, :net_suite_donor_export]
+  require_permission :can_create_organization?, only: [:net_suite_organizations_export]
```

Comment:
[Thought Only] This may not be the correct permission. At one point we made a reports role. I'm wondering if reports people need access to this. For now I think it's ok but we may want to verify permission to these with Lisa.